src/seedu/addressbook/data/person/Name.java

```diff
@@ -55,6 +55,14 @@ public boolean equals(Object other) {
                 && this.fullName.equals(((Name) other).fullName)); // state check
     }
 
+    /**
+     * Returns true of the other name is very similar to this name.
+     * Two names are considered similar if ...
+     */
+    public boolean isSimilar(Name other) {
```

Comment:
as mentioned in class, the first step should be writing a test for this method. the test will fail (can't even compile), but it's part of TDD process. You then write minimal code to pass this new test.